openapi/src/main/java/io/micronaut/openapi/visitor/ConfigUtils.java

```diff
@@ -1193,6 +1195,33 @@ private static PropertyNamingStrategies.NamingBase toJacksonStrategy(String nami
         };
     }
 
+    public static JsonInclude.Include getInclude(VisitorContext context) {
+        var value = getConfigProperty(MICRONAUT_OPENAPI_PROPERTY_INCLUDE, context);
+        if (value == null) {
+            return null;
+        }
+        return toJacksonInclude(value.toUpperCase(Locale.ENGLISH), context);
+    }
+
+    private static JsonInclude.Include toJacksonInclude(String include, VisitorContext context) {
+        if (include == null) {
+            return null;
+        }
+        return switch (include.toUpperCase(Locale.ENGLISH)) {
+            case "ALWAYS" -> JsonInclude.Include.ALWAYS;
+            case "NON_NULL" -> JsonInclude.Include.NON_NULL;
+            case "NON_ABSENT" -> JsonInclude.Include.NON_ABSENT;
```

Comment:
Do't neeed to add not supported values. micronaut-openapi can correctly interpret only `NON_NULL` and `ALWAYS`